tests/code/test_volunteers.py

```diff
@@ -109,3 +110,26 @@ def test_updateEventParticipants():
 
     with pytest.raises(DoesNotExist):
         EventParticipant.get(EventParticipant.user=="agliullovak", EventParticipant.event==3)
+
+@pytest.mark.integration
+def test_backgroundCheck():
+
+    updatebackground = setUserBackgroundCheck("khatts","CAN",False)
+    updatedModel = list(BackgroundCheck.select().where(BackgroundCheck.user == "khatts",BackgroundCheck.type == "CAN"))
+    assert updatedModel[0].passBackgroundCheck == False
+
+    updatebackground = setUserBackgroundCheck("khatts","FBI",True)
+    updatedModel = list(BackgroundCheck.select().where(BackgroundCheck.user == "khatts",BackgroundCheck.type == "FBI"))
+    assert updatedModel[0].passBackgroundCheck == True
+
+    updatebackground = setUserBackgroundCheck("khatts","SHS",False)
+    updatedModel = list(BackgroundCheck.select().where(BackgroundCheck.user == "khatts",BackgroundCheck.type == "SHS"))
+    assert updatedModel[0].passBackgroundCheck == False
+
+    updatebackground = setUserBackgroundCheck("neillz", "FBI",False)
+    updatedModel = list(BackgroundCheck.select().where(BackgroundCheck.user == "neillz",BackgroundCheck.type == "FBI"))
+    assert updatedModel[0].passBackgroundCheck == False
+
+    updatebackground = setUserBackgroundCheck("mupotsal","SHS",True)
+    updatedModel = list(BackgroundCheck.select().where(BackgroundCheck.user == "mupotsal",BackgroundCheck.type == "SHS"))
+    assert updatedModel[0].passBackgroundCheck == True
```

Comment:
since you are interested in one result every time, I would use `.get` instead of `.select` and `list` and `[0]`

Reply:
fixed